config/initializers/simple_form.rb

```diff
@@ -118,7 +118,7 @@
   # in this configuration, which is recommended due to some quirks from different browsers.
   # To stop SimpleForm from generating the novalidate option, enabling the HTML5 validations,
   # change this configuration to true.
-  config.browser_validations = false
+  config.browser_validations = true
```

Comment:
q oq to vendo é, q ta ativando as validações aqui e desabilitando uma a uma nos inputs